### PR TITLE
Couple of miscellaneous cleanups

### PR DESCRIPTION
**Pull request by jcsp**
...of things that appear to be unused.

### CODE REVIEW

File: pkg/operator/operator.go [hunk 1]
```diff
@@ -94,12 +93,6 @@ func (o *Operator) Run() error {
 		<-time.After(initRetryDelay)
 	}
 
-	rookAgent := agent.New(o.context.Clientset)
```
Comment (travisn) [inline pkg/operator/operator.go]: The rook agent is needed to help orchestrate the kubernetes volume plugins with mount, attach, etc. Why remove it?

Comment (jcsp) [reply pkg/operator/operator.go]: When I grepped the code I couldn't see this instance in the operator being used anywhere?  (the instantiation in cmd/rook/agent.go obviously makes sense, this one seemed dead).

I'm a newbie to this codebase though so I have a feeling you're about to explain to me what I've missed :-)

Comment (jbw976) [reply pkg/operator/operator.go]: This definitely looks in use to me still @jcsp, please do not remove this.

This here calls this `Run()` function: https://github.com/rook/rook/blob/master/cmd/rook/operator.go#L79

Comment (jcsp) [reply pkg/operator/operator.go]: Ah, I get it now -- this class isn't the agent, it's a component of the operator that goes and runs the agent elsewhere.

**Comment by jcsp** [comment]
Removed the agent-related commit (got confused about the two distinct `agent` classes in the codebase)

**Comment by jbw976** [comment]
Thanks for following up and for this contribution @jcsp!

**Pull request merged**